apps/frontend/assets/src/omg-alert/reducer.js

```diff
@@ -270,6 +270,12 @@ export const alertsReducer = createReducer([], {
       createAlertState('Submitted transaction to Ethereum successfully.', 'success')
     ]
   },
+  'WEB3/DEPOSIT/SUCCESS': state => {
```

Comment:
Should it be `WEB3/DEPOSIT/SUCCESS` if there's still the metamask approval step needed? Other success cases seem to be the completely successful one. Maybe `WEB3/DEPOSIT/ASK_APPROVAL`? Not sure.

Reply:
ah thats a good point. will update to make this more accurate 👍